src/app/components/team/team.component.html

```diff
@@ -0,0 +1,14 @@
+<article>
+    <img src="assets/pictures/{{ team.picture }}" alt="{{ team.name }}" />
+    <h3 >{{ team.name }}</h3>
```

Comment:
no h2 and no h1 on the page, consider using some into the about us page component
OR
change those h3 into h2 and add a h1 into about us component

Reply:
ok